### PR TITLE
Issue #5253 - Modified REST Connector feature description

### PR DESCRIPTION
**Pull request by navidsh**
PR for Issue #5253

### CODE REVIEW

**Comment by navidsh** [comment]
#build

**Comment by navidsh** [comment]
#build

File: dev/com.ibm.websphere.appserver.features/visibility/public/restConnector-2.0/resources/l10n/com.ibm.websphere.appserver.restConnector-2.0.properties [hunk 1]
```diff
@@ -14,7 +14,8 @@
...
 #
 
-description=A secure admin connector that can be used locally or remotely using any JDK. It enables \
-remote access by clients via a REST-based connector and requires SSL and basic user security configuration. \
-Client support includes JMX. This feature supersedes the restConnector-1.0 feature and does not include the \
+description=A secure REST-based admin connector that can be used locally or remotely using any JDK. It enables \
+remote access from a Java client or directly through an HTTPS call. It is protected by a single administrator \
+role and requires SSL to keep the communication confidential. The HTTPS port used by the connector are defined \
+through the default httpEndpoint. This feature supersedes the restConnector-1.0 feature and does not include the \
 jaxrs-1.1 feature.
```
Comment (bsbyrd1) [inline dev/com.ibm.websphere.appserver.features/visibility/public/restConnector-2.0/resources/l10n/com.ibm.websphere.appserver.restConnector-2.0.properties]: Suggestion:

A secure, REST administrative connector that can be used locally or remotely using any Java developer kit. This connector enables remote access from a Java client or directly through an HTTPS call. It is protected by a single administrator role and requires SSL to keep the communication confidential. The HTTPS port used by the connector is defined through the default HTTP endpoint. This feature is superseded by the restConnector-2.0 feature and enables the jaxrs-1.1 feature.

Question: How does the user know which feature is being described here?

**Comment by navidsh** [comment]
@bsbyrd1 Thanks for your feedback. I updated the text. This text will show up in an auto generated KC article: https://www.ibm.com/support/knowledgecenter/en/SSEQTP_liberty/com.ibm.websphere.liberty.autogen.nd.doc/ae/rwlp_feature_restConnector-2.0.html

**Pull request merged**